bkrdoc/analysis/linter/output_generator.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
+__author__ = 'blurry'
+
+from bkrdoc.analysis.parser import bkrdoc_parser
+from bkrdoc.analysis.linter import linter
+
+class OutputGenerator(object):
+    file_ = ""
+    parser = ""
+    # phases = []
+
+    def __init__(self, file):
+        self.parser = bkrdoc_parser.Parser(file)
+        self.parser.parse_data()
+
+    def analyse(self):
+
+        #for elem in self.parser.argparse_data_list:
+        #    print(elem)
+
+        _linter = linter.Linter()
```

Comment:
:D out of words? Please try to find other name for _linter because _ means that variable _linter is protected so you have to have a reason why to have it protected. Otherwise it is just mystification.